.changeset/honest-apricots-fail.md

```diff
@@ -2,4 +2,67 @@
 '@shopify/cli-hydrogen': patch
 ---
 
-Added `h2 upgrade` command
+# New `h2 upgrade` command
+
+We are introducing a new hydrogen cli `upgrade` command that:
+
+- Makes upgrading hydrogen as easy as typing `h2 upgrade` in your terminal
+- Provides a quick summary of the key `features` and `fixes` of any available
+  hydrogen version(s)
+- Generates a `TODO` instructions file detailing all cumulative code changes required
+  to adopt a given hydrogen version
+- Provides a gentle notice during development detailing when a hydrogen is outdated, as well as a quick glance into the number of hydrogen version available
+
+## Basic use
```

Comment:
I think the flags explanation and alternative usage can just go to the docs instead of the changeset? Most people will just run `h2 upgrade` directly in the project folder without flags. Maybe `--dry-run` could be mentioned but the others might be too much?